src/adapter/src/coord/sequencer.rs

```diff
@@ -1799,6 +1799,8 @@ impl<S: Append + 'static> Coordinator<S> {
         self.catalog_transact(Some(session), ops, |_| Ok(()))
```

Comment:
Would it make sense to have another failpoint before `self.catalog_transact` and have the test run for a 3-rd time with it?

Reply:
I think if the crash happens before, the operation is not run at all. There seems to be another related issue (see Joe's comment https://github.com/MaterializeInc/materialize/pull/16150#issuecomment-1320209907) that this might surface, but I'd favor adding the failpoint together with a test that fixes an issue.